libs/core/src/scss/themes/_colors.scss

```diff
@@ -169,10 +185,24 @@ $main-colors: map.remove(
   @return color.mix(white, $color, $weight * 1%);
 }
 
+@function _set-tint-value($color-name, $color) {
+  @if map.has-key($shade-tint-colors, $color-name) {
+    @return map.get($shade-tint-colors, $color-name);
+  }
+  @return _get-tint-value($color);
+}
```

Comment:
The newly introduced functions is essentially just a variation of _getting_ the value, not setting it. Can we extend the existing functions instead, and then pass the parameter by name where used?

```
@function _get-tint-value($color, $weight: 10, $color-name) {
  @if map.has-key($shade-tint-colors, $color-name) {
    @return map.get($shade-tint-colors, $color-name);
  }
  @return color.mix(white, $color, $weight * 1%);
}
```